### PR TITLE
Add 8.9 notes about IP usability improvements

### PR DESCRIPTION
**Pull request by 6hundreds**

### CODE REVIEW

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -74,6 +75,22 @@ or
 
 > Calculating task graph as configuration cache cannot be reused because file '.../some-file.txt' **has been replaced by a directory**.
 
+<a name="ip-usability"></a>
+
+### Isolated Projects usability improvements
+
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:81]:
```suggestion

The [Isolated Projects](userguide/isolated_projects.html) feature isolates the configuration models of projects from each other such that the build logic of one project cannot directly access the mutable state of another project.

```

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -74,6 +75,22 @@ or
...
+Depending on a task from another project in string-notated form is a common idiom:
+
+```
+foo.dependsOn(":a:bar")
+```
+
+Starting with this release, this is no longer considered a violation of Isolated Projects boundaries.
+
+#### `gradle init` generates Isolated Projects compatible projects
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:92]:
```suggestion
#### `gradle init` generates Isolated Projects compatible projects

```

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -74,6 +75,22 @@ or
...
+[Build Init Plugin](userguide/build_init_plugin.html) supports multimodule project layout. Starting with this release, `gradle init` generates projects compatible with Isolated Projects restrictions.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:93]:
```suggestion
The [Build Init Plugin](userguide/build_init_plugin.html) supports creating multi-module projects. 
Starting with this release, `gradle init` also generates projects compatible with Isolated Projects restrictions.
```

**Review by lkasso — CHANGES_REQUESTED** [review]

File: platforms/documentation/docs/src/docs/release/notes.md (hunk 3 above)
Comment (6hundreds) [reply platforms/documentation/docs/src/docs/release/notes.md:93]: @lkasso This `also` word is a bit confusing IMO. There is no option to generate project either compatible with IP or not. Should we omit it?

**Review by lkasso — APPROVED** [review]

**Review by bamboo — COMMENTED** [review]
👍

**Pull request merged**